pywinauto/win32functions.py

```diff
@@ -103,13 +103,15 @@
 ShowWindow			= 	ctypes.windll.user32.ShowWindow
 GetWindowContextHelpId =	ctypes.windll.user32.GetWindowContextHelpId
 GetWindowLong		=	ctypes.windll.user32.GetWindowLongW
+GetWindowLong.argtypes = [win32structures.HWND, ctypes.c_int]

```

Comment:
We can rely on built-in Python definitions in `ctypes.wintypes` for some types:
```python
from ctypes import wintypes
wintypes.HWND
```

Reply:
Meanwhile I just follow the same convention as the rest of the signatures in this module. Let's keep it consistent. I think, we already talked that we'll have a separate commit to switch to definitions from ctypes

Reply:
OK, we can fix it separately. I've found it's issue #650.